src/cli.rs

```diff
@@ -8,6 +8,12 @@ pub struct Args {
     /// Enabled by default if the app is running under `cargo leptos ...`
     #[arg(short = 'n', long = "no-open", default_value_t = in_cargo_leptos())]
     pub no_open: bool,
+    ///This flag enables the TRACE and DEBUG level log.
+    #[arg(long = "vv", default_value_t = in_cargo_leptos())]
+    pub vv: bool,
+    ///This flag enables the DEBUG level log.
+    #[arg(short = 'v', long = "verbose", default_value_t = in_cargo_leptos())]
+    pub verbose: bool,
```

Comment:
Why _two_ flags? You don't have to implement this by hand, as `clap` already supports it. See here for an example,

https://github.com/DeterminateSystems/nix-installer/blob/329861feca73c00362bbe5004eb30fdb92ba398b/src/cli/arg/instrumentation.rs#L32-L34